### ISSUE TITLE
stile/react doesn't seem to be included in the published dist/

### ISSUE THREAD

**Issue by kewah** [bloodyowl/stile#1]
Hej,

I've just installed `stile` via npm and `stile/react/` doesn't seem to be included in the published `dist/`.
See the current `dist/index.js`: https://gist.github.com/kewah/59db2cc7c2be4d383709

Therefore, when I try to import `ResizeProvider` I get an error:

```
import { ResizeProvider, resizeable } from 'stile/react'

> Error: Cannot find module 'stile/react'
```

Thanks

**Comment by bloodyowl** [comment]
fixed in 0.1.3

**Comment by bloodyowl** [comment]
see https://github.com/bloodyowl/stile/releases/tag/0.1.3
